bentoml/_internal/frameworks/pytorch.py

```diff
@@ -1,48 +1,53 @@
+from __future__ import annotations
+
 import typing as t
 import logging
-from typing import TYPE_CHECKING
+import functools
 from pathlib import Path
 
 import cloudpickle
-from simple_di import inject
-from simple_di import Provide
 
 import bentoml
 from bentoml import Tag
 
-from ..models import PT_EXT
-from ..models import SAVE_NAMESPACE
+from ..models import Model
 from ..utils.pkg import get_pkg_version
+from ...exceptions import NotFound
 from ...exceptions import BentoMLException
-from .common.pytorch import torch
-from .common.pytorch import BasePyTorchRunner
-from .common.pytorch import PyTorchTensorContainer  # pylint: disable=unused-import
-from ..configuration.containers import BentoMLContainer
+from ..models.model import ModelContext
+from ..models.model import ModelSignaturesType
+from .common.pytorch import torch  # type: ignore
+from .common.pytorch import PyTorchTensorContainer
 
-if TYPE_CHECKING:
-    from ..models import ModelStore
+__all__ = ["load_model", "get_runnable", "get", "PyTorchTensorContainer"]
```

Comment:
why do we need this here? we should only set `__all__` at the top level

Reply:
This is the standard way to get linter to stop complaining about A `imported but not used`